### DISCUSSION TITLE
Phrase de déconnexion

### DISCUSSION THREAD

**Discussion by maximpouliot** [MTESSDev/FRW#116] [Annonces]
Bonjour, 

est-ce que la phrase de déconnexion est personnalisable? Nous aimerions dire au client qu'il peut retourner a son dossier? 

Pour le moment c'est : Vous êtes maintenant déconnecté

Merci!

**Comment by LarcM7** [comment]
Bonjour, présentement ce n'est pas personnalisable, mais vous pouvez en faire la demande si vous le souhaitez via l'ouverture d'une issue s'il vous plaît.
Merci

**Comment by maximpouliot** [comment]
Fait merci beaucoup!
